### ISSUE TITLE
responsibleProducer, safetyInformation - czy ma być ustawiany dla każdej oferty?

### ISSUE THREAD

**Issue by tomylee0** [allegro/allegro-api#10212]
Nie bardzo rozumiem zasób /sale/responsible-producers. 

Czy każdy sprzedający ma mieć swoją prywatną listę producentów? 
Czy wystawiając ofertę za każdym razem sprzedający musi uzupełnić w ofercie productSet.responsibleProducer? 
Czy to znaczy, że ten sam produkt u dwóch sprzedających może mieć inny responsibleProducer? 

Te same pytania dotyczą również safetyInformation. Wydawać by się mogło, że responsibleProducer oraz safetyInformation powinny być w bazie produktowej w produkcie. Ale struktura zasobów sugeruje co innego.

**Comment by MaciejFrackowiak** [comment]
Tak, zasób ten - /sale/responsible-producers pozwala sprzedającemu na stworzenie własnej bazy producentów. Jest ona dedykowana per sprzedający. I tak, każdy sprzedający produkt może dodać własny podmiot jako responsible producer.
Docelowo, będziemy chcieli produkty z naszej bazy wzbogacić o te dane. Nie ma jednak możliwości, aby każdy jeden produkt na pewno miał uzupełnioną tę wartość, dlatego też zasoby zostały oddane, aby sprzedający mógł we własnym zakresie określić odpowiedni podmiot.
W przypadku informacji o bezpieczeństwie produktu to również na sprzedającym spoczywa obowiązek określenia, czy produkt oferowany przez niego ma instrukcję, czy też jej nie posiada.

**Comment by tomylee0** [comment]
Dzięki wielkie za szybką odpowiedź. Wydaje się być to teraz jasne ale dopytam jeszcze dla pewności. Czyli jeśli sprzedaję książki i mam tysiąc wydawnictw (producentów) to teraz jako sprzedający powinienem zebrać dane dotyczące tysiąca producentów (nazwa, adres, telefon) i wstawić je do swojego zasobu /sale/responsible-producers by móc je potem podpinać pod ofertę?

Dopytuję ponieważ nie jest mi znana publicznie dostępna taka baza producentów która pozwoliłaby na automatyczne wypełnienie wspomnianego zasobu. A ręczne wyszukiwanie i wpisywanie tysiąca producentów wydaje się być czynnością dosyć nieefektywną i uciążliwą.

**Comment by MaciejFrackowiak** [comment]
Co do zasady tak, ponieważ odpowiedzialność za prezentację danych spoczywa na sprzedającym zgodnie z regulacją UE.

**Comment by tomylee0** [comment]
Dzięki. Czy coś już wiadomo na temat powiązania pola responsibleProducer z parametrem odpowiedzialnym za producenta, np. parametr "id": "223541" dla "category": { "id": "66794" }? Parametr jest planowany do usunięcia czy pozostanie niezależnie? Lub może istnieje już lub jest planowana walidacja zgodności?

**Comment by MaciejFrackowiak** [comment]
W tej kwestii nie mam żadnej informacji jakoby miało dojść do takiego "matchowania".
